Sources/Components/SuggestShippingView/ShippingRequestErrorView.swift

```diff
@@ -0,0 +1,66 @@
+import Combine
+import FinniversKit
+
+public final class ShippingRequestErrorView: UIView {
+    private var viewModel: ShippingRequestErrorViewModel!
+    private var cancellables = Set<AnyCancellable>()
+    private let containerView = UIStackView(axis: .vertical)
+
+    private lazy var titleLabel: Label = {
+        let label = Label(style: .captionStrong)
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        return label
+    }()
+
+    private lazy var messageLabel: Label = {
+        let label = Label(style: .caption)
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        return label
+    }()
+
+    private lazy var readMoreButton = Button(style: .flat)
+
+    public static func create(with viewModel: ShippingRequestErrorViewModel) -> ShippingRequestErrorView {
+        let view = ShippingRequestErrorView()
+        view.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
We should use this initializer instead, since we have it for free :) 

```suggestion
        let view = ShippingRequestErrorView(withAutoLayout: true)
```